Datagrid/Datagrid.php

```diff
@@ -126,15 +126,15 @@ public function buildPager()
             $this->formBuilder->add($filter->getFormName(), $type, $options);
         }
 
-        $this->formBuilder->add('_sort_by', 'hidden');
+        $this->formBuilder->add('_sort_by', 'Symfony\Component\Form\Extension\Core\Type\HiddenType');
```

Comment:
You understand well @WouterJ : do the test cover this part? Can either of you both please fix it ? Just like in `CollectionType` ?